docs/release.md

```diff
@@ -28,27 +28,25 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 1. Update Contrib to use the latest in development version of Core by running `make update-otel` in Contrib root directory. This is to ensure that the latest core does not break contrib in any way. If it results in any changes, submit a PR to Contrib.
    -  🛑 **Do not move forward until this PR is merged.**
 
-2. Determine the version number that will be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
+1. Determine the version number that will be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
```

Comment:
I don't mind moving to numbering like this since it's easier to edit, but could we make this change in a separate PR? Including it in this one makes it harder to review.

Reply:
sure, I can do that.

Reply:
fixed